launcher/README.md

```diff
@@ -16,29 +16,14 @@ To run the notifier that pops up if `/proc/uptime` (how long the system has been
 
 ## Developer environment
 
-The next version of Qubes will include PyQt5 in dom0, which is why we also support PyQt5.
-
-### PyQt4 instructions
-
-In both cases, make sure to install the appropriate `python3-pyqt{4,5}` package in Debian Buster.
+Install the `python3-pyqt5` package in Debian Bullseye.
 The use of system-site-packages in the virtualenv config will import it.
 
 To run the preflight updater outside of `dom0`:
 
 1. `cd securedrop-workstation/launcher`
-2. `sudo apt install python3-pyqt4`
+2. `sudo apt install python3-pyqt5`
```

Comment:
```suggestion
2. `sudo apt install make python3-venv python3-pyqt5`
```

Reply:
I guess there's the question of whether we just refer them to the workstation/"setting up sd-dev" docs, where a lot of these dependencies are already prerequisites. But I'll add these in :)

Reply:
Mmm, good point. I (too?) would favor more self-sufficient instructions were practical 👍